rdt-open-reader/src/covidIndonesia/res/values-in/strings.xml

```diff
@@ -39,4 +39,7 @@
     <string name="str_page_info">Halaman {0} dari {1}</string>
     <string name="json_form_error_msg">Ditemukan %d kesalahan pada form. Mohon benarkan untuk mengirim.</string>
     <string name="json_form_on_next_error_msg">Mohon benarkan kesalahan pada form untuk melanjutkan.</string>
+    <string name="onescan_is_not_installed">Aplikasi OneScan diperlukan untuk melanjutkan proses ini</string>
+    <string name="ok">Oke</string>
+    <string name="error">Kesalahan</string>
```

Comment:
We don't need to add translations here. They will be done later via Transifex.

Reply:
Done.